### PR TITLE
Set frame selection to current attachment when applicable

### PR DESCRIPTION
**Pull request by danielbachhuber**
See #350

### CODE REVIEW

File: js/build/shortcode-ui.js [hunk 1]
```diff
@@ -740,6 +740,24 @@ var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
 	_openMediaFrame: function(e) {
 		e.preventDefault();
 		this.frame.open();
+		if ( this.model.get( 'value' ) ) {
+			var query_args = {
+				'post__in': [ this.model.get( 'value' ) ],
+				'orderby': 'post__in',
+				'perPage': 1
+			};
+			var attachments = wp.media.query( query_args );
```
Comment (danielbachhuber) [inline js/build/shortcode-ui.js]: @goldenapples This is producing 0 attachments:

image

Any idea why?

Comment (goldenapples) [inline js/build/shortcode-ui.js]: Not sure. I spent a while digging through the parameters available to `wp.media.query`, but I can't seem to make any of them actually work:

screen shot 2015-08-13 at 11 04 54 am

I'll come back to this later, but I'm not getting it right now...

Comment (danielbachhuber) [inline js/build/shortcode-ui.js]: Dang. I'll take another look at it.

Comment (danielbachhuber) [inline js/build/shortcode-ui.js]: https://github.com/alleyinteractive/wordpress-fieldmanager/pull/379 might work instead

**Pull request merged**